ext/readline/readline.c

```diff
@@ -262,6 +266,10 @@ PHP_FUNCTION(readline_info)
 		add_assoc_long(return_value,"pending_input",rl_pending_input);
 		add_assoc_string(return_value,"prompt",SAFE_STRING(rl_prompt));
 		add_assoc_string(return_value,"terminal_name",(char *)SAFE_STRING(rl_terminal_name));
+		completion_append_str[0] = (char)rl_completion_append_character;
+		completion_append_str[1] = '\0';
+		add_assoc_string(return_value,"completion_append_character",completion_append_str);
```

Comment:
Similarly, this line could be changed to `add_assoc_str(...,..., ZSTR_CHAR(rl_completion_append_character));`, also saving the conditional define above.